core/frontend/src/components/version-chooser/VersionChooser.vue

```diff
@@ -596,6 +602,28 @@ export default Vue.extend({
       }
       return this.available_versions.local.some((image) => image.sha === sha)
     },
+    get_input_file(): File | null {
+      const { files } = document.getElementById('file') as HTMLInputElement
+
+      if (files !== null && files.length > 0) {
+        return files[0]
+      }
+
+      return null
```

Comment:
```suggestion
      if (files == null || files.length <= 0) {
          return null
      }
      
      return files[0]
```

just so the main/intended flow is obvious.

Reply:
Changed to the suggested!